web-api/src/persistence/postgres/cases/getCasesMetadataWithCounselByLeadDocketNumber.ts

```diff
@@ -9,16 +9,16 @@ export const getCasesMetadataWithCounselByLeadDocketNumber = async ({
   applicationContext: ServerApplicationContext;
   leadDocketNumber: string;
 }): Promise<RawCase[]> => {
-  const dbCases = await getDbReader(reader =>
+  const dbCaseData = await getDbReader(reader =>
```

Comment:
Slightly more rigorous than "anecdotal": this seems to be speeding up `getCaseByDocketNumber` for consolidated cases by 1+ seconds for some of the cases I have been experimenting with. (15631-22, 16017-21, 16017-22, 12235-20.)






[3:17](https://flexion.slack.com/archives/C07BG6FUSP5/p1742498233596919)
I just know its consolidated because its Amgen but can’t actually view it in test :woman-shrugging::skin-tone-3:
[3:17](https://flexion.slack.com/archives/C07BG6FUSP5/p1742498253926429)
16017-21 also times out (edited)